backend/handler/question_groups.go

```diff
@@ -98,3 +97,49 @@ func (s *Server) PostQuestionGroup(e echo.Context, params PostQuestionGroupParam
 
 	return e.JSON(http.StatusCreated, res)
 }
+
+func (s *Server) PutQuestionGroup(e echo.Context, questionGroupID QuestionGroupId, params PutQuestionGroupParams) error {
+	user, err := s.repo.GetOrCreateUser(*params.XForwardedUser)
+
+	if err != nil {
+		e.Logger().Errorf("failed to get or create user: %v", err)
+
+		return echo.NewHTTPError(http.StatusInternalServerError, "Internal server error")
+	}
+
+	if !user.IsStaff {
+		return echo.NewHTTPError(http.StatusForbidden, "Forbidden")
+	}
+
+	var req PutQuestionGroupJSONRequestBody
+
+	if err := e.Bind(&req); err != nil {
+		e.Logger().Errorf("failed to bind request body: %v", err)
+
+		return echo.NewHTTPError(http.StatusBadRequest, "Bad request")
+	}
+
+	var updateQuestionGroup model.QuestionGroup
```

Comment:
`res`には`id`と`questions`が入っていません（更新自体は正常にできていて、GETリクエストを送ると`id`と`question`が入ったデータが返ってくるのは確認しました）
`copier.Copy`で`req`→`updateQuestionGroup`→`res`とコピーしていて、最初の`req`に`id`と`question`が入っていないからだと思います。`id`と`question`はここで編集できてはいけないので`req`に入っていないのが正しいですが、レスポンスには含まれていてほしいです。
空の`model.QuestionGroup`を用意するのではなく、既存のものに対して`copier.Copy`をするのが良いと思います（更新があった所だけ上書きされます）
あと、動詞から始まる名前だと関数っぽいので、単に`questionGroup`とするか`updatedQuestionGroup`とすると良いかも 👀 
```suggestion
	updateQuestionGroup, err := s.repo.GetQuestionGroup(uint(questionGroupID))
	
	if err != nil {
		if errors.Is(err, model.ErrNotFound) {
			return echo.NewHTTPError(http.StatusNotFound, "Not found")
		}

		e.Logger().Errorf("failed to get question group: %v", err)
		return echo.NewHTTPError(http.StatusInternalServerError, "Internal server error")
	}
```